pages/services/spinnaker/0.3.1-1.9.2/overview/servergroups/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: layout.pug
+navigationTitle: Server Groups
+excerpt: Using Spinnaker applications and server groups
+title: Applications and Server Groups
+menuWeight: 6
+model: /services/spinnaker/data.yml
+render: mustache
+---
+
+# Concepts
+
+The following three concept pictures are from Will Gorman's great [Mesoscon presentation](
```

Comment:
Again, no need for "great". Please delete.

Reply:
Deleted.